tests/e2e/performance/test_vdbench_workload.py

```diff
@@ -135,31 +136,20 @@ class TestVDBenchWorkload(E2ETest):
     The total number of files is calculating by : files * (width ** depth)
     """
 
+    @pytest.mark.parametrize('template', [
+        pytest.param("VDBench-Basic.yaml", marks=pytest.mark.workloads()),
```

Comment:
suggestion:
you can import 'workloads' in line #17:
```python
from ocs_ci.framework.testlib import E2ETest, workloads
```
and here do:
```python
pytest.param("VDBench-Basic-FS.yaml", marks=workloads)
```

Reply:
please address

Reply:
I changed it.